### DISCUSSION TITLE
Should we set secrets.OPENAI_API_KEY?

### DISCUSSION THREAD

**Discussion by dandansamax** [camel-ai/camel#153] [Q&A] [answered]
Now the pytest in the github workflow does not work because of missing OPENAI_API_KEY. Should we set that to enable PRs auto-merging?

**Comment by lightaime** [comment]
It is set as a GitHub secrete currently.

**Reply by lightaime** [reply, marked as answer]
The `OPENAI_API_KEY` can be accessed by the team members through newing a branch instead of creating a fork for PRs.

**Reply by dandansamax** [reply]
I see. Thank you so much for your help!
